Arrowgene.Ddon.Database/Files/Database/Script/migration_mail_sqlite.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS "ddon_system_mail" (
```

Comment:
These files need to be preprocessed similar to our previous SQL scheme, Otherwise, some of this syntax will not work (like `AUTOINCREMENT`) in other databases. 

From an academic point of view, I read using `IF NOT EXISTS` in a migration is not considered good practice. This means we don't understand the state of the database we are updating.